appengine-jetty-managed-runtime/pom.xml

```diff
@@ -247,6 +243,18 @@
               </resources>
             </configuration>
           </execution>
+          <execution>
+            <phase>clean</phase>
+            <goals>
+              <goal>removeImage</goal>
+            </goals>
+            <configuration>
+              <imageName>appengine-mvn-opensource</imageName>
+              <imageTags>
+                <imageTag>latest</imageTag>
```

Comment:
Removing latest is not guaranteed to remove the artefacts created by this build/branch.
In development, if multiple branches are being developed it may well be likely that java 7 and java 8 images are both deployed,   Deleting the latest only deletes the last built and is not related to this poms version in any way.